test/e2e/storage/persistent_volumes-local.go

```diff
@@ -356,6 +357,96 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("should support SELinux labeling on local storage volume", func() {
+
+		var testVol *localTestVolume
+
+		BeforeEach(func() {
+			testVol = setupLocalVolumePVCPV(config, node0)
+		})
+
+		AfterEach(func() {
+			cleanupLocalVolume(config, testVol)
+		})
+
+		It("should be able to mount volume in two pods at the same time, write from pod1, and read from pod2 when using the same SELinux label", func() {
+			By("Creating podSc1")
+			podSc1, podSc1Err := createSecPod(config, testVol, false, false, &v1.SELinuxOptions{
+				Level: "s0:c0,c1",
+			})
+			Expect(podSc1Err).NotTo(HaveOccurred())
+
+			framework.ExpectNoError(framework.WaitForPodRunningInNamespace(config.client, podSc1))
```

Comment:
It calls `CreateSecPod`, which I see is waiting for the pod to be running [here](https://github.com/kubernetes/kubernetes/pull/49749/files/a6ca584d523f80dfa72aad99abeb016f4b7c71f0#diff-420013911dac4a3b0f16035b6e90a571R920)